content/doc/pipeline/index.adoc

```diff
@@ -158,8 +158,11 @@ build branches that contain a `Jenkinsfile`
 
 === More Information
 
-As with any Jenkins plugin you can install the Pipeline plugin via the Plugin
-Manager in a running Jenkins instance. To explore Pipeline without installing
+As with any Jenkins plugin, you can install the Pipeline plugin using the Plugin
+Manager in a running Jenkins instance. No Pipeline plugin installation is needed for Jenkins 2,
```

Comment:
This is not correct. Jenkins 2 doesn't bundle the Pipeline plugins, but merely suggests/encourages that the user install them.